docs/guides/modules/about-circleci/pages/introduction-to-the-circleci-web-app.adoc

```diff
@@ -153,12 +153,9 @@ The following settings are available in the project settings page. If you do not
 
 **Groups**: Provides a list of user groups that have been granted project-specific permissions. For more information, see the xref:permissions-authentication:manage-groups.adoc[Manage groups] page.
 
-**Pipelines**: Provides a list of pipelines that have been set up for the project, or the option to begin setting up pipelines if none exist.
+**Project Setup**: Provides a list of pipelines that have been set up for the project and their associated triggers. See xref:orchestrate:pipelines.adoc[Pipelines Overview and Setup] for more information. This page includes the option to begin setting up pipelines if none exist. The setup UI for scheduled pipelines is located either here or in the *Triggers* section (if available for your integration). To set up a xref:orchestrate:scheduled-pipelines.adoc[scheduled pipeline] you will define a timetable, parameter, and attribution to automatically run a pipeline when the criteria is met.
 
-**Triggers**: Depending on your integration type, the triggers page shows either:
-
-* A list of configured xref:orchestrate:pipelines.adoc#pipelines-and-triggers[pipeline triggers]
-* Set up UI for scheduled pipelines if available for your integration. To set up a xref:orchestrate:scheduled-pipelines.adoc[scheduled pipeline] you will define a timetable, parameter, and attribution to automatically run a pipeline when the criteria is met.
+*Pipelines* and *Triggers*: Some integration types will see two pages in the sidebar for setting up and managing pipelines and triggers. The options available in these pages are equivalent to those available in the *Project Setup* page.
```

Comment:
Here too, I think I'd go for something like this, but happy for you to keep the existing one (or a version of ) if you think it's better
```suggestion
*Pipelines* and *Triggers*: These pages let you set up pipelines and triggers when using the Gitlab, Bitbucket Cloud and Bitbucket Data Center integrations. 
```